library/Icinga/Application/Hook/LoginButton/LoginButton.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/* Icinga Web 2 | (c) 2025 Icinga GmbH | GPLv2+ */
+
+namespace Icinga\Application\Hook\LoginButton;
+
+class LoginButton
+{
+    /**
+     * @var string
+     */
+    protected $logoUrl;
+
+    /**
+     * @var string
+     */
+    protected $label;
+
+    /**
+     * @var callable
+     */
+    protected $onSuccess;
+
+    /**
+     * Create an additional button to be displayed below the login form
+     *
+     * @param callable $onSuccess What to do if the button is pressed
+     * @param string $label Text to show on the button
+     * @param ?string $logoUrl URL to an image to show on the button before the text
+     */
+    public function __construct(callable $onSuccess, string $label, ?string $logoUrl = null)
```

Comment:
Please make the class `readonly`  and use constructor property promotion.

Reply:
If I think about it a little longer, we could make this more flexible by providing `ValidHtml $content` and `?Attributes $attributes` instead of `label` and `logoUri`. Then it would be possible to provide either images or icons, as well as other important attributes such as `title`. What do you think?